### PR TITLE
YAMLLintBear: Change `SEVERITY_LEVEL`

### PR DESCRIPTION
**Pull request by Techievena**
Change the severity level of the syntax errors detected from
normal to major.

Fixes https://github.com/coala/coala-bears/issues/1525

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [ ] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [ ] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

**Comment by supergr35** [comment]
Rebase and I believe travis should pass.

File: bears/yaml/YAMLLintBear.py [hunk 1]
```diff
@@ -1,12 +1,15 @@
...
 import yaml
 
 
 @linter(executable='yamllint',
         output_format='regex',
         output_regex=r'.+:(?P<line>\d+):(?P<column>\d+): '
-                     r'\[(?P<severity>error|warning)\] (?P<message>.+)')
+                     r'\[(?P<severity>error|warning)\] (?P<message>.+)',
+        severity_map={'error': RESULT_SEVERITY.MAJOR,
+                      'warning': RESULT_SEVERITY.MAJOR})
```
Comment (adhikasp) [inline bears/yaml/YAMLLintBear.py]: I think `warning` should get mapped to `RESULT_SEVERITY.NORMAL`.

Comment (Techievena) [inline bears/yaml/YAMLLintBear.py]: My bad

File: tests/yaml/YAMLLintBearTest.py [hunk 2]
```diff
@@ -83,3 +90,23 @@
...
+
+    def setUp(self):
+        self.section = Section('')
+        self.uut = YAMLLintBear(self.section, Queue())
+
+    def test_warning(self):
+        content = test_file.splitlines()
+        with prepare_file(content, None) as (file, fname):
+            with execute_bear(self.uut, fname, file) as results:
+                self.assertEqual(results[6].severity, RESULT_SEVERITY.NORMAL)
```
Comment (NiklasMM) [inline tests/yaml/YAMLLintBearTest.py]: I find it very hard to understand what's going on here. Can you please add comments explaining which errors these test cases detect?

Comment (NiklasMM) [inline tests/yaml/YAMLLintBearTest.py]: Maybe make individual minimal files with one result each, instead of using the same big file.

Comment (Techievena) [inline tests/yaml/YAMLLintBearTest.py]: @NiklasMM I have checked for the `result.message` now which will make clear about which messages are `error` and which are `warning`. I think using additional files will unnecessarily elongate the code.

File: tests/yaml/YAMLLintBearTest.py [hunk 3]
```diff
@@ -83,3 +90,27 @@
...
+
+    def setUp(self):
+        self.section = Section('')
+        self.uut = YAMLLintBear(self.section, Queue())
+
+    def test_warning(self):
+        content = test_file.splitlines()
+        with prepare_file(content, None) as (file, fname):
+            with execute_bear(self.uut, fname, file) as results:
+                self.assertEqual(results[6].message, 'comment not indented'
```
Comment (NiklasMM) [inline tests/yaml/YAMLLintBearTest.py]: This is much better! I suggest you also add an assertion on the line of the result to make it extra clear!

**Comment by NiklasMM** [comment]
Code looks good now. Just one thing about the commit message. The `error` category in yamllint doesn't seem to contain only syntax errors. For example "too many blank lines" is classified as error, but surely no syntax error. Maybe you can just remove the word "syntax" :D

**Comment by NiklasMM** [comment]
ack fe7b2f4

**Comment by NiklasMM** [comment]
@rultor merge

**Pull request merged**